### ISSUE TITLE
mustache-interpolation-spacing not enforcing exact number of spaces when set to "always"

### ISSUE THREAD

**Issue by jblotus** [vuejs/eslint-plugin-vue#214]
**Tell us about your environment**
centos vm

* **ESLint Version:** 4.8.0
* **eslint-plugin-vue Version:** 3.13.1
* **Node Version:** v6.9.2

**What did you do? Please include the actual source code causing the issue.**
Was trying to implement mustache-interpolation-spacing to enforce exactly one space between curly brackets

Sorry for the hastily prepared report, but looking at the tests for this rule I don't see the case I am talking about covered. I think it is an important one though since 1 space seems to be the idiomatic way based on the official docs.

```js
   {
      filename: 'test.vue',
      code: '<template><div>{{   text   }}</div></template>',
      output: '<template><div>{{text}}</div></template>',
      options: ['never'],
      errors: [{
        message: 'Found 3 whitespaces, 1 expected.',
        type: 'Identifier'
      }, {
        message: 'Found 3 whitespaces, 1 expected.',
        type: 'VExpressionEnd'
      }]
    }

```

**Comment by jblotus** [comment]
https://github.com/vuejs/eslint-plugin-vue/pull/215

**Comment by michalsnik** [comment]
As pointed by @mysticatea in #215 this is intended behaviour, and it makes it aligned with equivalent eslint core rules.
